template/partials/css.js

```diff
@@ -6,7 +6,51 @@ module.exports = function (results, hx) {
   <style>
     body {
       font-family: 'Roboto', sans-serif;
+      background: white;
+      margin: 0px;
     }
+    .object {
+      margin: 0 0 15px 0;
+    }
+    .title{
+      background: #1C8DC6;
```

Comment:
Could this background be changed to `#2C3E50`, and make the color of the text white? Thanks